index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
With `double` there's always the option to go unit-less (which I think we've done elsewhere here? but it escapes me). However, I suppose it's best to use what users expect, which seems to be millimeters if this is focal length, and my brief internet search wasn't way off. I know the TAG [recommends units like milliseconds](https://w3ctag.github.io/design-principles/#milliseconds) for time, but doesn't say anything about distance. Fyi.